praetorian_cli/ui/console/commands/marcus.py

```
@@ -58,8 +58,10 @@ def _cmd_marcus(self, args):
             self.context.mode = 'query'
 
         self.console.print(f'[primary]Entering conversation mode[/primary] [dim](type "/back" to return)[/dim]')
-        self.console.print(f'[dim]Commands: /back, /new, /query, /agent, or just chat[/dim]')
+        self.console.print(f'[dim]Commands: /back, /new, /query, /agent, /skill <path>, /skills, /unskill <name>, or just chat[/dim]')
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)